content/docs/user-guide/gems/reference/terrain/_index.md

```diff
@@ -24,3 +24,9 @@ To enable the Terrain Gem, do the following:
 
 1. Use **O3DE Project Manager** or the command line to add the Terrain Gem to your project.
 2. Build your project using Project Manager, Visual Studio, or CMake.
+
+## Components
+
+| Component | Description | 
+| - | - |
+| [Layer Spawner](/docs/user-guide/gems/reference/terrain/layer_spawner.md) | Controls the bounds of a terrain layer, and allows prioritization of multiple layers. |
```

Comment:
Since the display name of the component is "Terrain Layer Spawner", we should call it that here instead of just "Layer Spawner".

This component actually causes the terrain layer to spawn, so for description, what about something like this?
"Spawns a region of terrain within the given bounds and controls prioritization of multiple overlapping terrain layers."